src/basics/GlobalStyles.js

```diff
@@ -37,6 +37,11 @@ export const GlobalStyles = createGlobalStyle`
   twitter-widget {
     margin: auto;
   }
+  ::-webkit-scrollbar {
```

Comment:
Verrrry nervous about globally hiding scrollbars. I notice that the entire page scrolls a little bit, I think there's some extra height coming in from the sidebar

Reply:
I looked at this earlier but hadn't committed a fix, I think `SideNavContainer` has the wrong height. It's doing `calc(100vh - 3.75rem)` but the logo section is 72px = 4.5rem.

Reply:
How about adding this to the JSON responses component and I will find the extra height?

Reply:
This could def make sense on those docs elements

Reply:
Updated!